core/parser.mly

```diff
@@ -871,6 +876,12 @@ block_contents:
 | exp                                                          { ([], $1) }
 | SEMICOLON | /* empty */                                      { ([], with_pos $loc (TupleLit [])) }
 
+bindings_block:
```

Comment:
I wonder whether it is worth to introduce a production that is used only once. Also, I find the name slightly misleading, since it doesn't produce a `Block`.